tests/components/enigma2/test_config_flow.py

```diff
@@ -145,5 +145,5 @@ async def test_form_import_errors(
             data=TEST_IMPORT_FULL,
         )
 
-    assert result["type"] == FlowResultType.FORM
-    assert result["errors"] == {"base": error_type}
+    assert result["type"] == FlowResultType.ABORT
```

Comment:
```suggestion
    assert result["type"] is FlowResultType.ABORT
```

And can we also test that we now have an issue created